README.md

```diff
@@ -1,60 +1,52 @@
 # gh - The GitHub CLI tool
 
-The #ce-cli team is working on a publicly available CLI tool to reduce the friction between GitHub and one's local machine for people who use the command line primarily to interact with Git and GitHub. https://github.com/github/releases/issues/659
+`gh` is GitHub on the command line. It brings pull requests, issues, and other GitHub concepts to
+the terminal next to where you are already working with `git` and your code.
 
-This tool is an endeavor separate from [github/hub](https://github.com/github/hub), which acts as a proxy to `git`, since our aim is to reimagine from scratch the kind of command line interface to GitHub that would serve our users' interests best.
+![screenshot](https://user-images.githubusercontent.com/98482/73207890-1b7f6180-410b-11ea-9c75-d20c436eba71.png)
 
-# Installation
+## Usage
 
-_warning, gh is in a very alpha phase_
+- `gh pr [status, list, view, checkout, create]`
+- `gh issue [status, list, view, create]`
+- `gh help`
 
-## macOS
+Check out the [docs][] for more information.
 
-`brew install github/gh/gh`
-
-## Debian/Ubuntu Linux
 
-1. `sudo apt install git` if you don't already have git
-2. Download the `.deb` file from the [releases page](https://github.com/github/gh-cli/releases/latest)
-3. `sudo dpkg -i gh_*_linux_amd64.deb`  install the downloaded file
+## Comparison with hub
 
-_(Uninstall with `sudo apt remove gh`)_
+For many years, [hub][] was the unofficial GitHub CLI tool. `gh` is a new project for us to explore
```

Comment:
I'm a bit surprised to see `hub` called "unofficial" when `hub` is at `hub.github.com` where other projects don't have the authority to have webpages, `hub` is owned by the `github` organization, and `hub` [was announced on the GitHub blog in '09](https://github.blog/2009-12-10-hub-git-with-github/). I wonder if you can clarify the different expectations users might have out of an official CLI?